tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -459,3 +467,76 @@ def test_should_respond_404(self, test_client):
         )
         assert response.status_code == 404
         assert response.json()["detail"] == "The Asset with uri: `s3://bucket/key` was not found"
+
+
+class TestQueuedEventEndpoint:
+    default_time = "2020-06-11T18:00:00+00:00"
+
+    @pytest.fixture(autouse=True)
+    def setup(self) -> None:
+        clear_db_assets()
+
+    def teardown_method(self) -> None:
+        clear_db_assets()
+
```

Comment:
most of [these](https://github.com/apache/airflow/pull/43934/files#diff-5d24f2ed749e37ef9f12eff9a54ad581a0a23df847421f62e9e0289060e931a6R475-R508) fixture and method we already have [here](https://github.com/astronomer/airflow/blob/c593a3f86c400738a743b3739608ed7e95a462dc/tests/api_fastapi/core_api/routes/public/test_assets.py#L122). We can reuse them

Reply:
Good catch, sounds good. I refactored around a bit in tests